src/components/ModalFlows/CreateProposalScreens/ActionFees.js

```diff
@@ -51,98 +51,56 @@ function ActionFeesModal({ agreement, onCreateTransaction }) {
         </span>
       </Field>
 
-      {compactMode && (
-        <div
-          css={`
-            margin-top: ${GU * 2}px;
-            weight: 100%;
-          `}
-        >
-          <img
-            css={`
-              height: ${GU * 4}px;
-              width: ${GU * 4}px;
-            `}
-            src={iconFees}
-            alt=""
-          />
-        </div>
-      )}
-
       <div
         css={`
           display: flex;
           width: 100%;
           height: ${GU * 4}px;
           margin-top: ${!compactMode ? GU * 3 : 0}px;
+          align-items: center;
         `}
       >
-        {!compactMode && (
-          <div
-            css={`
-              flex: 0.25;
-            `}
-          >
-            <img
-              css={`
-                height: ${GU * 4}px;
-                width: ${GU * 4}px;
-              `}
-              src={iconFees}
-              alt=""
-            />
-          </div>
-        )}
+        <img
+          css={`
+            height: ${GU * 4}px;
+            width: ${GU * 4}px;
+          `}
+          src={iconFees}
+          alt=""
+        />
 
-        <div
+        <h3
           css={`
-            flex: ${compactMode ? 1 : 2};
-            padding: 0 0 0 ${!compactMode ? 1.25 * GU : 0}px;
-            align-self: center;
+            font-weight: 600;
+            margin-left: 3%;
           `}
         >
-          <h3
-            css={`
-              font-weight: 600;
-            `}
-          >
-            Action Deposit
-          </h3>
-        </div>
-        <div
+          Action Deposit
+        </h3>
+
+        <span
           css={`
-            flex: 0.5;
-            align-self: center;
-            ${compactMode ? textStyle('body4') : textStyle('body3')};
+            text-align: 'left';
+            font-weight: 600;
+            position: absolute;
```

Comment:
Ah lets not use abolsute if we don't need to.

If we want to separate the dollarAmount and token amount from the icon and text we can do so by encapsulating them in a div and in the parent div use `justify-content: space-between`.

A code snippet of how it would look like would be

```
<div
  css={`
    width: 100%;
    height: ${GU * 4}px;
    margin-top: ${!compactMode ? GU * 3 : 0}px;
    display: flex;
    align-items: center;
    justify-content: space-between;
  `}
>
  <div
    css={`
      display: flex;
      align-items: center;
    `}
  >
    <img
      css={`
        height: ${GU * 4}px;
        width: ${GU * 4}px;
      `}
      src={iconFees}
      alt=""
    />

    <h3
      css={`
        font-weight: 600;
        margin-left: ${1 * GU}px;
      `}
    >
      Action Deposit
    </h3>
  </div>
  <div>
    <span
      css={`
        text-align: left;
        font-weight: 600;
        ${compactMode ? textStyle('body3') : textStyle('body2')};
        margin-right: ${(compactMode ? 1.5 : 3) * GU}px;
      `}
    >
      ${dollarAmount}
    </span>

    <span
      css={`
        font-weight: 600;
        ${compactMode ? textStyle('body3') : textStyle('body2')};
      `}
    >
      {formatedAmount} {token.symbol}
    </span>
  </div>
</div>

```

Reply:
Hey @fabriziovigevani thanks for the clarification on this, i've just applied the changes.
Also the GU pattern was my bad, i'll take that in mind for the future :)

Reply:
No worries! Thanks for updating them!